app/code/Magento/InventoryInStorePickupApi/etc/webapi.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
+    <route url="/V1/inventory/in-store-pickup/get-nearby-pickup-locations" method="POST">
```

Comment:
Ok, my last comment was an overkill. We definitely don't want to get data with POST request. 

Sorry for the confusion, idea was to note it as a point for additional investigation, nothing else.

Reply:
Yes, sorry I was took this point too literally.
I have changed method back to "GET".